dev/idm-am-common-user/idm/conf/repo.ds.json

```diff
@@ -0,0 +1,489 @@
+{
+    "embedded": false,
+    "maxConnectionAttempts" : 5,
+    "security": {
+        "keyManager": "jvm",
+        "trustManager": "jvm"
+    },
+    "ldapConnectionFactories": {
+        "bind": {
+            "connectionSecurity": "none",
+            "connectionPoolSize": 50,
+            "heartBeatIntervalSeconds": 60,
+            "heartBeatTimeoutMilliSeconds": 10000,
+            "primaryLdapServers": [
+                {
+                    "hostname": "configstore-0.configstore",
+                    "port": 1389
+                }
+            ],
+            "secondaryLdapServers": [ ]
+        },
+        "root": {
+            "inheritFrom": "bind",
+            "authentication": {
+                "simple": {
+                    "bindDn": "cn=Directory Manager",
+                    "bindPassword": "password"
```

Comment:
This probably should use property substitution. I need to update the oauth2 sample also.